data-raw/scenario_geography_x_ald_sector.R

```diff
@@ -47,6 +47,51 @@ scenario_geography_x_ald_sector <- tibble::tribble(
   "Russia",      "Power",     "WEO2019_NPS",
```

Comment:
I would suggest sorting this by scen_geo for easier understanding. You could easily generated respective tibble with datapasta

Reply:
done